components/ProfileLessons.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import '../scss/profileLessons.scss'
```

Comment:
@coltonehrman thoughts on importing this scss file into our main scss file?

Reply:
this is specific to this file tho.

Reply:
We are doing this for other styles that are specific to the component too though ie challenge material (granted I worked on this)

Reply:
i mean unless we decide to stick with importing everything into main or not it doesnt really matter at this point. but i do think we should decide to do one or the other, not mixing it. so nobody gets confused

Reply:
Let's not import everything into main. We should import styles as we need it.

Reply:
but wouldnt it be fair to say we need every style? otherwise, that would mean we have useless scss files.plus in the end it'll all be bundled together the same basically right?

Reply:
It's fair to say that, but you don't need every style in every page / every component. This means you lose scoped scss functionality.

Reply:
so i believe the way we are importing the scss files currently has no scope `import './style.scss`... i think you have to do `import styles from './style.scss` then reference it in the component code

Reply:
https://nextjs.org/docs/basic-features/built-in-css-support

Reply:
@coltonehrman `but wouldnt it be fair to say we need every style? otherwise, that would mean we have useless scss files.plus in the end it'll all be bundled together the same basically right?` The way I see it, keeping everything together makes things easier to track. So one example is... if I remove this component, I may forget to remove from `main.scss` and code reviewers wouldn't know. If you put the `scss` together in the component, then our css could be cleaner.